app/components/post_list/post_list.tsx

```diff
@@ -251,6 +291,14 @@ const PostList = ({
         return removeListener;
     }, []);
 
+    useEffect(() => {
+        setTimeout(() => {
+            listRef?.current?.scrollToOffset({
+                offset: 0,
+            });
+        }, 1000);
+    }, [listRef]);
```

Comment:
I guess this is only on mount.

two things:
1. no need for the reference as a dependency
2. this is taking one second, so you better cancel the timeout if the component get's unmounted.
3. Why 1s ?